### ISSUE TITLE
Error output does not format colors

### ISSUE THREAD

**Issue by christianalares** [jaredpalmer/razzle#1724]
### Current Behavior
If I produce an expected error in a react component the errors are not formatting correctly (see screenshot below)

This is my component which obviously will give an error since  `test.hello.world.thisDoesNotExist` doesn't exist:
```javascript
import React from 'react'

const test = {
  hello: 'world',
}

const Home = () => {
  return (
    <div>
      <h1>Home</h1>
      {test.hello.world.thisDoesNotExist}
    </div>
  )
}

export default Home
```

This is what the browser is showing:
![image](https://user-images.githubusercontent.com/893819/134654597-fcb5a0a5-3254-4bd0-9043-ca056815b152.png)

### Expected behavior
The browser should show the above error properly formated instead of all the `[0m [90m 8 | [39m`-characters.
Perhaps something like this (random googled image):
![image](https://user-images.githubusercontent.com/893819/134654881-c9ae6c9d-889b-4d04-9dca-750feb0f2efc.png)

### Reproducible example
```
npx create-razzle-app razzle-test
yarn start
```

* Edit a file to throw an error
* Look at the output in the browser

### Suggested solution(s)
I don't know how this is done and therefore not how to solve it but the problem speaks for itself.

| Software         | Version(s) |
| ---------------- | ---------- |
| Razzle           | ^4.1.0
| Razzle Plugins   | -
| Node             | v12.18.1
| Browser          | Google Chrome 93.0.4577.63
| npm/Yarn         | Yarn v1.22.11
| Operating System | macOS Big Sur 11.5.2
| TypeScript       | -
| React            | ^17.0.2

**Comment by fivethreeo** [comment]
This error is from the express server not webpack dev server

**Comment by christianalares** [comment]
> This error is from the express server not webpack dev server

I see. Is there a way to make this error to be formatted properly?

**Comment by fivethreeo** [comment]
Wish I knew, maybe some express middleware?

**Comment by christianalares** [comment]
Is this format made for std out? Because this error looks correct in the terminal (from the server). Would be great to see this kind of error in a nice way on the client as well.

**Comment by fivethreeo** [comment]
Yeah, I think so. But not really a razzle issue. But if it is possible in express somehow it should be doable.
